package.json

```diff
@@ -32,6 +32,7 @@
     "verify": "lerna run verify --stream",
     "version": "lerna clean --yes && lerna bootstrap --ignore-scripts -- --package-lock-only --no-audit && git add packages/*/package-lock.json"
   },
+  "dependencies": {},
```

Comment:
When you execute an uninstall via a test (e.g. `npm uninstall fastify`), npm has added this property.
Before I went with the NPM namespace solution (e.g. fastify2) I had to install and uninstall versions, which always gave me this package.json diff.

Now that we have a different solution, I still kept it here, because I thought it should somehow be part of the package.json?

But I can remove it again. I have no strong opinion here :)

Reply:
I don't have a strong opinion either, the empty dependencies section just looked a bit odd in the diff, that's all. Let's toss a coin, it really makes no difference :)

Reply:
I'll just remove it for now :)